hubmap_commons/hm_auth.py

```diff
@@ -477,6 +480,25 @@ def setProcessSecret(secret):
         if AuthCache.procSecret is None:
             AuthCache.procSecret = secret
 
+    @staticmethod
+    def setGlobusGroups(globusJson):
+        groupIdByName = {}
+        for group in globusJson:
+            if 'name' in group and 'uuid' in group and 'generateuuid' in group and 'displayname' in group and not string_helper.isBlank(
```

Comment:
The logic within this `setGlobusGroups()` is essentially the same as https://github.com/hubmapconsortium/commons/blob/globus-groups-changes/hubmap_commons/hm_auth.py#L518-L528 

Can you create a new internal function for code reuse?

Reply:
@yuanzhou Good catch, will do

Reply:
@yuanzhou This should be resolved in latest commit